kube/services/jobs/covid19-notebook-etl-cronjob.yaml

```diff
@@ -0,0 +1,54 @@
+# gen3 job run covid19-notebook-etl S3_BUCKET <S3_BUCKET>
```

Comment:
Not a blocker but will this run the job or the cronjob? If it's not the cronjob, we might need to change this comment.